plugins/jdg72/pom.xml

```diff
@@ -29,6 +29,16 @@
             <version.infinispan>8.5.0.ER3-redhat-1</version.infinispan>
          </properties>
       </profile>
+      <profile>
+         <id>demos</id>
```

Comment:
Sorry, this profile should go away as well. This was related to the dist-exec demo JAR file that was removed